.circleci/config.yml

```diff
@@ -122,6 +121,7 @@ jobs:
       - image: circleci/ruby:2.4
     working_directory: /tmp/deploy
     environment:
+      # TODO: Add support for deploying Bionic to Staging Unstable
```

Comment:
As I understand this could be removed and bionic is OK to add to the deployment list as you already covered the Packagecloud.sh changes

Reply:
Correct - will do.